frontend/src/components/Login.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+
+const SIGNUP_URL = 'https://project-auth-technigo.herokuapp.com/users' 
+const LOGIN_URL = 'https://project-auth-technigo.herokuapp.com/sessions'
+const SECRETS_URL ='https://project-auth-technigo.herokuapp.com/secrets'
+
```

Comment:
Great that you've created url-variables! Makes the code further down easy to read.